list.rb

```diff
@@ -0,0 +1,19 @@
+require './enumerable'
```

Comment:
- [ ] To import MyEnumerable please use require_relative which is used for files in your directory instead of require which is for files outside your directory, this is a ruby convention. "Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory."

Reply:
I appreciate the explanation @GzimAsani thank you very much 👏👏.